playbooks/common/openshift-master/config.yml

```diff
@@ -337,12 +361,54 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
-- name: Configure master instances
-  hosts: oo_masters_to_config
+- name: Configure first master instance
```

Comment:
Oh I think I see the problem you were alluding to now, how do I not pass openshift_version for first master, but then pass for everyone else if they're all handled by the same play after removing the duplication. Looking into your suggestions now.